package/rpm/victoriametrics.service

```diff
@@ -6,8 +6,9 @@ After=network.target
 Type=simple
 StartLimitBurst=5
 StartLimitInterval=0
-Restart=on-abort
-ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data start
+Restart=on-failure
+RestartSec=1
+ExecStart=/usr/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data
```

Comment:
Probably the binary name must be changed from `victoriametrics` to `victoria-metrics-prod` similar to `package/victoria-metrics.service`?